solutions/cf-920E.mdx

```diff
@@ -0,0 +1,232 @@
+---
+id: cf-920E
+source: CF
+title: Connected Components?
+author: Daniel Qiu
+---
+
```

Comment:
You're switching between inclusive first person (we) and second person (you) a lot through this solution. Can you stick to one so that it's more consistent? (I think we is prob better but up to u)